packages/engineer/test/dev-server.unit.ts

```diff
@@ -76,9 +78,15 @@ describe('engineer:dev-server', function () {
         dispose: () => Promise<void>;
         engine: RuntimeEngine;
         runtimeFeature: typeof devServerFeature;
+        outputPath: string | undefined;
         config: { featureName: string | undefined; port: number };
     }> => {
-        const { dispose, engine, devServerFeature } = await startDevServer({
+        const {
+            dispose,
+            engine,
+            outputPath: devOutputPath,
```

Comment:
why do we need to return the outputPath from this method?

Reply:
It's needed for the expected value, see line 617